simulation/behavior_tree_plugin/src/action_node.cpp

```diff
@@ -33,6 +33,18 @@
 
 namespace entity_behavior
 {
+BT::PortsList operator+(const BT::PortsList & ports_0, const BT::PortsList & ports_1)
+{
+  BT::PortsList ports;
+  for (const auto & [name, port_info] : ports_0) {
+    ports.try_emplace(name, port_info);
```

Comment:
Is there any reason to use try_emplace even though you're not doing any moves or forwarding?

Reply:
use insert insert of try_emplace.